src/flashbar/__tests__/collapsible.test.tsx

```diff
@@ -203,44 +205,26 @@ describe('Collapsible Flashbar', () => {
       const flashbar = renderFlashbar({ items: [sampleItems.error] });
       const list = findList(flashbar)!;
       expect(list).toBeTruthy();
-      const itemCounterElement = findCounterElement(flashbar);
-      expect(itemCounterElement).toBeUndefined();
+      const itemCounterElement = findOuterCounter(flashbar);
+      expect(itemCounterElement).toBeFalsy();
       expect(list.getElement()).not.toHaveAttribute('aria-describedby');
     });
 
     it('applies aria-describedby to the toggle button, referencing the item counter', () => {
       const flashbar = renderFlashbar();
-      const itemCounterElementId = findCounterElement(flashbar)!.id;
+      const itemCounterElementId = findOuterCounter(flashbar)!.id;
       expect(itemCounterElementId).toBeTruthy();
       const toggleButton = findInnerToggleButton(flashbar);
       expect(toggleButton).toHaveAttribute('aria-describedby', itemCounterElementId);
     });
 
-    it('announces updates to the item counter with a live region', () => {
-      const customLabels = {
-        errorCountAriaLabel: 'Custom error ARIA label',
-        successCountAriaLabel: 'Custom success ARIA label',
-        infoCountAriaLabel: 'Custom info ARIA label',
-        inProgressCountAriaLabel: 'Custom progress ARIA label',
-        warningCountAriaLabel: 'Custom warning ARIA label',
-      };
-      const flashbar = renderFlashbar({
-        i18nStrings: { ...customLabels },
-        items: [
-          sampleItems.success,
-          sampleItems.warning,
-          sampleItems.success,
-          sampleItems.warning,
-          sampleItems.error,
-          sampleItems.warning,
-        ],
-      });
-      const liveRegionElement = findOuterToggleElement(flashbar)!.querySelector('[aria-live="polite"]')!.parentElement;
-      expect(liveRegionElement).toHaveTextContent(`${customLabels.errorCountAriaLabel}: 1`);
-      expect(liveRegionElement).toHaveTextContent(`${customLabels.successCountAriaLabel}: 2`);
-      expect(liveRegionElement).toHaveTextContent(`${customLabels.warningCountAriaLabel}: 3`);
-      const ariaHiddenElement = liveRegionElement!.nextElementSibling;
-      expect(ariaHiddenElement).toHaveAttribute('aria-hidden', 'true');
+    it('announces updates to the item counter with aria-live', () => {
+      const flashbar = renderFlashbar();
+      const counter = findOuterCounter(flashbar);
+      expect(counter).toHaveAttribute('aria-live', 'polite');
+      // We add `role="status"` as well, to maximize compatibility
+      // https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions#roles_with_implicit_live_region_attributes
```

Comment:
(minor) This is good to know, but I would expect this comment in the component code, not the test code.